### PR TITLE
match real output

### PR DESCRIPTION
**Pull request by NargiT**
I was doing the "getting started" and I notice that the documentation does not give the same output as my cli.

### CODE REVIEW

File: content/en/docs/chart_template_guide/getting_started.md [hunk 1]
```diff
@@ -200,6 +202,8 @@ NAME: clunky-serval
 LAST DEPLOYED: Tue Nov  1 17:45:37 2016
 NAMESPACE: default
 STATUS: DEPLOYED
+REVISION: 1
+TEST SUITE: None
```
Comment (hickeyma) [inline content/en/docs/chart_template_guide/getting_started.md]: Comment above applies here also.

File: content/en/docs/chart_template_guide/getting_started.md [hunk 2]
```diff
@@ -117,6 +117,8 @@ NAME: full-coral
 LAST DEPLOYED: Tue Nov  1 17:36:01 2016
 NAMESPACE: default
 STATUS: DEPLOYED
+REVISION: 1
+TEST SUITE: None
```
Comment (hickeyma) [inline content/en/docs/chart_template_guide/getting_started.md]: Resources are no longer output in Helm 3.  Output is no as follows:
```console
$ helm install full-coral ./mychart
NAME: full-coral
LAST DEPLOYED: Mon Mar  2 10:09:29 2020
NAMESPACE: default
STATUS: deployed
REVISION: 1
TEST SUITE: None

```

The following sentence is no longer valid : `In the output above, we can see that our ConfigMap was created. `

File: content/en/docs/chart_template_guide/getting_started.md [hunk 3]
```diff
@@ -117,6 +117,8 @@ NAME: full-coral
...
 
 RESOURCES:
```
Comment (NargiT) [inline content/en/docs/chart_template_guide/getting_started.md]: Ok I will remove this part also. I though it was only there to display what happends on the cluster and was not related to helm output. Will do this tonight. Thx for the feedback

Comment (hickeyma) [reply content/en/docs/chart_template_guide/getting_started.md]: Great, thanks

**Pull request merged**